### PR TITLE
CVSL-1911: Remove unnecessary code.

### PR DESCRIPTION
**Pull request by justin-gardiner**

### CODE REVIEW

**Review by nishanthmahasamudram — APPROVED** [review]
LGTM

File: src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/caseload/ComCaseloadService.kt [hunk 1]
```diff
@@ -222,13 +221,12 @@ class ComCaseloadService(
           LicenceStatus.TIMED_OUT,
         )
       }
-      validLicenceStatus == true && !offender.nomisRecord.isBreachOfTopUpSupervision()
```
Comment (nishanthmahasamudram) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/caseload/ComCaseloadService.kt:225]: Are we not checking **is breach of top up supervision case** anymore??

Comment (justin-gardiner) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/caseload/ComCaseloadService.kt:225]: Yes it's already done by EligibilityService before we get here

Comment (nishanthmahasamudram) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/caseload/ComCaseloadService.kt:225]: oh you are directly calling **getIneligibilityReasons**.
Ok.

**Pull request merged**